inductiva/tasks/task.py

```diff
@@ -178,3 +178,21 @@ def get_execution_time(self) -> float:
                                                 "%Y-%m-%dT%H:%M:%S.%f+00:00")
 
         return (end_time - start_time).total_seconds()
+
+    def get_machine(self) -> str:
```

Comment:
Just to add a bit more: Shall we also provide the ram and number of cores? This is not essential I think for the our current purpose, but it might be also useful for the users.